tests/test.rs

```diff
@@ -459,3 +459,19 @@ fn asm_flags() {
     test.cmd(1).must_have("--abc");
     test.cmd(2).must_have("--abc");
 }
+
+#[test]
+fn gnu_apple_darwin() {
+    for (arch, version) in &[("x86_64", "10.7"), ("aarch64", "11.0")] {
```

Comment:
A comment that this is expected to break after https://github.com/rust-lang/rust/pull/104385 would be appreciated.

Reply:
I found `__set_env` and used that to override the version used, so this now should be immune to future minimum support bumps. Does that work instead?